src/NCPackageSelector.cc

```diff
@@ -1065,6 +1067,16 @@ void NCPackageSelector::replaceFilter( FilterMode mode)
 	   repoPopup->setKeyboardFocus();
 	   break;
         }
+	case Services:
+	{
+	   YTableHeader *hhh = new YTableHeader ();
```

Comment:
NP: I would have named that variable `header`, not `hhh`, that would make it better readable. But it's still ok.